src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML</IncludeInPackage>
```

Comment:
As mentioned previously, we can't be putting Mkl into the `Microsoft.ML` package.